src/sentry/static/sentry/app/components/contextPickerModal.tsx

```diff
@@ -202,6 +202,20 @@ class ContextPickerModal extends React.Component<Props> {
 
     this.navigateIfFinish([{slug: organization}], [{slug: value}]);
   };
+  get headerText() {
+    const {needOrg, needProject} = this.props;
+    if (needOrg && needProject) {
+      return t('Select an organization and a project to continue');
+    }
+    if (needOrg) {
```

Comment:
Small nit: I guess this is stylistic but do we do if/else if/else if/ else?

Reply:
@NisanthanNanthakumar I believe it is preferred to just return a value instead of using if/else.